nservicebus/azure-service-bus/message-lock-renewal.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Azure Service Bus Transport Message lock renewal
+summary: Extending message lock renewal
```

Comment:
summary is used in the context of being a proper sentences. eg it can be multiple sentences. so to be full grammatically correct and consistence the sentence should end in a full stop